README.md

```diff
@@ -2,6 +2,8 @@
 
 Let's show the world how to learn XQuery. Pull requests welcome. Quality over comprehensiveness - just the best, most helpful resources, please. Talk about [`#learnxquery`](https://twitter.com/search?q=learnxquery) on Twitter.
 
+[XQuery Update facility for the impatient, A quick introduction to the XQuery Update Facility](http://www.xmlmind.com/tutorials/XQueryUpdate/index.html), Xavier Franc. Presents the XQuery Update facilities introduced by the W3C recommendation.
```

Comment:
Thanks! Could you please move this into an appropriate section, such as "tutorials"?